DTS-STN/Life-Journeys
# Lj 90 accordion

# Description

[LJ-90] New accordion component 

the component is for future use.
might change the content to children instead of a prop.  
also I fixed the topic component key error 

## Test Instructions

1. Get the branch add the component to the index page or another page and see it working 

## Meets Definition of Done



[LJ-90]: https://lj-decd.atlassian.net/browse/LJ-90
